app/scripts/controllers/swaps.js

```diff
@@ -449,22 +460,22 @@ export default class SwapsController {
     this.store.updateState({ swapsState: { ...swapsState, routeState } });
   }
 
-  setSwapsLiveness(swapsFeatureIsLive) {
+  setSwapsLiveness(swapsLiveness) {
```

Comment:
The removal of the `swapsFeatureIsLive` param, along with spreading the new param onto state, was a bit confusing when I first read through this. It might be good to make it obvious which properties are getting set on state by the `swapsLiveness` parameter. Also, with this change, any property in the object passed to `setSwapsLiveness` will be written to state.

It might be good to destructure `swapsLiveness` and explictly write the `swapsFeatureIsLive` and `useNewSwapsApi` properties in the `updateState` call below.

Reply:
That's great feedback! I will explicitly use these 2 props when updating state to make it clearer and safer.